### PR TITLE
Initial unify build

### PR DESCRIPTION
**Pull request by psortos**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by psortos** [comment]
@conda-forge/staged-recipes ready for review/merge!

File: recipes/unify/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 8ddce812b2457212b7598fe574c9e6eb3ad69710f445391338270c7f8a71723c
+
+build:
+  number: 0
```
Comment (xhochy) [inline recipes/unify/meta.yaml:13]:
```suggestion
  number: 0
  noarch: python
```

File: recipes/unify/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  entry_points:
+    - unify = unify:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+    - untokenize
```
Comment (xhochy) [inline recipes/unify/meta.yaml:22]: This is probably not needed at installation time.
```suggestion
```

File: recipes/unify/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  run:
+    - python
+    - untokenize
+
+test:
+  commands:
+    - unify --help
```
Comment (xhochy) [inline recipes/unify/meta.yaml:29]:
```suggestion
test:
  requires:
    - pip
  commands:
    - pip check
    - unify --help
```

**Review by xhochy — COMMENTED** [review]
You should try using https://github.com/conda-incubator/grayskull to generate these recipes. All review comments should be automatically addressed by using it.

**Pull request merged**